android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -969,6 +969,17 @@ sealed class TranslationsAction : BrowserAction() {
         override val tabId: String,
         val pageSettings: TranslationPageSettings?,
     ) : TranslationsAction(), ActionWithTab
+
+    /**
+     * Sets the never translate sites.
+     *
+     * @property tabId The ID of the tab the [EngineSession] that requested the list.
+     * @property neverTranslateSites The never translate sites.
```

Comment:
nit: Maybe a bit more detail like `The list of sites that the user has opted to never translate.` or something.

Reply:
Updated the property comment to be more detailed